packages/scoped-custom-element-registry/test/ShadowRoot.test.html.js

```diff
@@ -292,6 +292,34 @@ describe('ShadowRoot', () => {
       });
     });
 
+    describe('createElementNS', () => {
```

Comment:
We also need tests in the "with custom registry" suite, yes?

Reply:
Oops, done.